content/docs/command-reference/commit.md

```diff
@@ -279,5 +279,5 @@ $ dvc status
 Data and pipelines are up to date.
 ```
 
-Nothing special is required, we simply `commit` to both the SCM and DVC. Since
-this pipeline is up to date, `dvc repro` will not do anything.
+Instead of reproducing the pipeline for changes that do not produce a different
```

Comment:
I think it should be ` produce different results` or ` produce a different result`.